supabase/migrations/101_anime_visibility_and_studio_names.sql

```
@@ -0,0 +1,107 @@
+    FOREIGN KEY (source, source_key)
+        REFERENCES public.studio_source_records(source, source_key)
+        ON DELETE CASCADE
+);
+
+CREATE INDEX IF NOT EXISTS studio_name_aliases_source_idx
+    ON public.studio_name_aliases (source, source_key);
+
+ALTER TABLE public.studio_source_records ENABLE ROW LEVEL SECURITY;
+ALTER TABLE public.studio_name_aliases ENABLE ROW LEVEL SECURITY;
+
+CREATE POLICY "studio source records: public can read open sources"
```

SQLFluff (4.3.0)

[error] Do not use special characters in identifiers.

(RF05)